src/query/const_score_query.rs

```diff
@@ -0,0 +1,126 @@
+use std::collections::BTreeMap;
+use std::fmt;
+
+use crate::query::{ConstScorer, Explanation, Query, Scorer, Weight};
+use crate::{DocSet, Score, Searcher, SegmentReader, TantivyError, Term};
+
+/// `ConstScoreQuery` is a wrapper over a query to provide a constant score.
+/// It can avoid unnecessary score computation on the wrapped query.
+///
+/// The document set matched by the `ConstScoreQuery` is strictly the same as the underlying query.
+/// The configured score is used for each document.
+pub struct ConstScoreQuery {
+    query: Box<dyn Query>,
+    score: Score,
+}
+
+impl ConstScoreQuery {
+    /// Builds a const score query.
+    pub fn new(query: Box<dyn Query>, score: Score) -> ConstScoreQuery {
```

Comment:
I think the `ConstScoreQuery` symbol is not accessible from client in the library.

Can you re-export it from `tantivy::query`?